src/main/java/core/basesyntax/dao/BetDaoImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Bet;
+
+import java.util.List;
+
+//  class for work with DB and table of bets
```

Comment:
better remove all comments in the repository

Reply:
Fixed.